stdlib/public/Concurrency/Task.cpp

```diff
@@ -470,6 +470,32 @@ const void *AsyncTask::getResumeFunctionForLogging() {
   return reinterpret_cast<const void *>(ResumeTask);
 }
 
+JobPriority swift::swift_task_currentPriority(AsyncTask *task)
+{
+  // This is racey but this is to be used in an API is inherently racey anyways.
+  auto oldStatus = task->_private().Status.load(std::memory_order_relaxed);
+  return oldStatus.getStoredPriority();
+}
+
+JobPriority swift::swift_task_basePriority(AsyncTask *task)
+{
+  JobPriority pri = task->_private().BasePriority;
+  SWIFT_TASK_DEBUG_LOG("Task %p has base priority = %zu", task, pri);
+  return pri;
+}
+
+static inline bool task_priority_is_unspecified(JobPriority priority) {
```

Comment:
You can just call this `isUnspecified` or something.  The `swift_feature_function_name` pattern is specifically for ABI entrypoints, or at least internal API to the `.swift` code.